### ISSUE TITLE
MultiMC / Prism Launcher source

### ISSUE THREAD

**Issue by Pandapip1** [lutris/lutris#5747]
Pretty self-explanitory: I'd love to be able to run my prism launcher instances using lutris. Bonus points if I can download modpacks too!

**Comment by v3xed-v0id** [comment]
if we are gonna do this we should also add a source for ATLauncher

**Comment by Pandapip1** [comment]
> if we are gonna do this we should also add a source for ATLauncher

That seems like a separate feature request to me. I'll open a new issue for that.

**Comment by v3xed-v0id** [comment]
okie thank u :3

**Comment by legluondunet** [comment]
This repository is not a place for requesting support with a game, runner or an installer.
If you have issues installing a game or launching it, make sure you follow our essential guides on [graphics driver installation](https://github.com/lutris/docs/blob/master/InstallingDrivers.md) and [dependencies for the wine runner](https://github.com/lutris/docs/blob/master/WineDependencies.md).
If you followed the guides and the issues persist, try asking for help in our official [Lutris Discord Server](https://discord.com/invite/Pnt5CuY).

**Comment by beedell-roke** [comment]
@legluondunet, [how] *is* this an example of what you describe?

**Comment by legluondunet** [comment]
Hello @RokeJulianLockhart , it is possible I didn't understand correctly this ticket, it is about script installer for running minecraft instances right? and install mods too? If not can you explain me more in details?

**Comment by beedell-roke** [comment]
@legluondunet, you are correct that the user could script the invocation of specific instances via the CLI. However, the issue author is requesting that Lutris expose Prism Launcher's instance list:

<p></p><img width="636" height="356" alt="Image" src="https://github.com/user-attachments/assets/48651de2-a4a3-43f5-bc7a-dae909adc37f" /><p></p>

…because that means that they would not need to utilise Prism's launcher. However, to me, it appears to be a significant scope creep, and I can't say that I understand why one wouldn't merely utilise Prism's launcher for this.

**Comment by legluondunet** [comment]
I must admit that, not being a Minecraft player, I don't quite understand the request. 
That said, it's perfectly possible to install multiple instances of a game with Lutris, then rename them to distinguish them.
And indeed, I just checked on the official "Prism Launcher" website, and Prism could very well meet the author's needs.
As it stands, this request does not appear to be directly related to the functioning of Lutris.

**Comment by Aphlait** [comment]
Prism is a launcher for multiple different instances of Minecraft. The original poster most likely wants Lutris to be able to import and run each instance as if they were different games. Lutris it seems is more about being a launcher that facilitates  the compatibility layers required for games to run in Linux. Personally, I switched from Windows to Linux very recently and I am missing the ability that Playnite had to import Prism Launchers instances as if they were separate games. However, I do understand if that is out of the scope of Lutris.

**Comment by kccricket** [comment]
@legluondunet Prism (and MultiMC and ATLauncher) is a launcher that helps the user manage "instances" of Minecraft. Each instance can be set to a different version, with a different mod loader, with different mods installed, running under different versions of Java. Each instance is essentially a different game. Playnite handles this by considering Prism a "source", in the same way that it considers Steam or Epic a source.

I believe the request here is for Lutris to support Prism as a game source where the instances in Prism appear as launchable games in Lutris. Prism is a fork of MultiMC which I believe is a fork of another one, so one integration should generally be similar for each member of the lineage.

If I take a look at building this integration, would you consider accepting a PR?

**Comment by legluondunet** [comment]
@strycore this ticket needs your attention, could you answer to @kccricket
